Makefile

```diff
@@ -135,7 +135,7 @@ upgrade: requirements check_pins ## update the requirements/*.txt files with the
 requirements.js: ## install JS requirements for local development
 	npm ci
 
-requirements: requirements.js ## install development environment requirements
+requirements: ## install development environment requirements
```

Comment:
why did you remove this?

Reply:
Great catch... ah...my bad ... remove during testing/debugging